build/credscan-exclusion.json

```diff
@@ -44,6 +44,26 @@
     {
       "file": "CryptographyTests.cs",
       "_justification": "Password contained within this file is not used for any production application/code. It is used as a means to access a self signed certificate used only for testing."
-    }
+    },
+{
+      "file": "AdalUniversalTestApp.Windows_TemporaryKey.pfx",
+      "_justification": "Legitimate UWP app certificate file with private key required for the app to function. This is sample test app."
+    },
+    {
+      "file": "valid_cert.pfx",
+      "_justification": "Self-signed certificate used by unit tests."
+    },
+    {
+      "file": "valid_cert2.pfx",
+      "_justification": "Self-signed certificate used by unit tests."
+    },
+    {
+      "file": "invalid_cert.pfx",
```

Comment:
seems like some of these are duplicates. are all needed?

Reply:
fixed